.zenodo.json

```diff
@@ -0,0 +1,79 @@
+{
+    "license": "Apache-2.0", 
+    "description": "<p>Expected to be the last version before the merge of the GLUE2.1 effort.</p>\n\n<p>Two new packages have been introduced to automatically install the required dependencies (including <code>cloud-info-provider</code> package):</p>\n\n<ul>\n\t<li>RPM: <code>cloud-info-provider-openstack</code>, deb: <code>python-cloud-info-provider-openstack</code></li>\n\t<li>RPM: <code>cloud-info-provider-opennebula</code>, deb: <code>python-cloud-info-provider-opennebula</code></li>\n</ul>\n\n<p>Full changelog:</p>\n\n<ul>\n\t<li>Integrate badges from shields.io and coveralls.io. (Alvaro Lopez)</li>\n\t<li>Improve and cleanup configuration files. (Enol Fernandez)</li>\n\t<li>Import a Jenkinsfile to handle SQA on Jenkins at IFCA. (Pablo Orviz)</li>\n\t<li>Fix UTF-8 output for OpenNebula. (Ruben Diez)</li>\n\t<li>Introduce CMF-specific meta-packages to deploy required dependencies. (Baptiste Grenier)</li>\n\t<li>Allow filtering private flavours. (Baptiste Grenier)</li>\n\t<li>Improve community health following discussions with Bruce Becker and al. (Baptiste Grenier)</li>\n\t<li>Use travis to lint, test, build and upload packages. (Baptiste Grenier)</li>\n</ul>", 
+    "language": "eng", 
+    "title": "EGI-Foundation/cloud-info-provider: 0.10.0", 
```

Comment:
So the issue that I can see is that here the json file will be added post tag creation, and the tag won't include it. After this will we have to maintain the json file manually, as part of the release preparation process?

Reply:
we could do something similar to what we do with the spec?

The source of truth should be spec, right - from that, we write the file name and the json.

Reply:
Yes, sounds good. Just need to find a way to be sure (how-to release doc? automatic tool?) that we won't forget to do it.